tests/collections/asr/test_asr_local_attn.py

```diff
@@ -0,0 +1,40 @@
+# Copyright (c) 2022, NVIDIA CORPORATION.  All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import pytest
+import torch
+
+from nemo.collections.asr.models import ASRModel
+
+
+class TestASRLocalAttention:
+    @pytest.mark.with_downloads()
+    @pytest.mark.unit
+    def test_forward(self):
```

Comment:
Please added another test to change the self attention, then save in first temp dir, move .nemo to another dir, delete first temp dir, and restore in second dir. 
Assert that the changes made were reflected in the model - you may need to update the model.cfg inside of nemo/collections/asr/parts/mixins/mixins.py

Reply:
Added